releasenotes/notes/trace-propagation-style-extract-order-baa0637bc4f506f2.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    The specification of ``DD_TRACE_PROPAGATION_STYLE_EXTRACT`` requires the configured styles be evaluated in order. However, the evaluation order had been hard-coded and so ignored the configured ordering.
```

Comment:
can we say "fixed" versus "hard-coded".

Also we're not actually saying what is fixed here. Maybe something like:

"The specification of ``DD_TRACE_PROPAGATION_STYLE_EXTRACT`` now respects the configured styles evaluation order. The evaluation order had previously been fixed and so the configured order was ignored."